### PR TITLE
Feat/replace value with iri as string or implicit to string

### PR DESCRIPTION
**Pull request by pmcb55**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@inrupt/solid-react-components",
-  "version": "0.5.3",
+  "version": "0.5.4",
```
Comment (NSeydoux) [inline package.json:3]: The version change should be featured in its own commit, in order to be able to tag it.

File: src/lib/classes/notification.js [hunk 2]
```diff
@@ -461,7 +461,7 @@ export class Notification {
          * Validate the list of notification IRIs and return a list of valid ones
          */
         const coreNotificationShape =
-          'https://shexshapes.inrupt.net/public/notifications/app-notification.shex';
+          'https://shexshapes.inrupt.net/public/notifications/core-notification.shex';
```
Comment (NSeydoux) [inline src/lib/classes/notification.js:464]: I recall we had a discussion about this in a previous PR, but I don't remember the conclusion we reached. Should this change be kept or reverted ?

File: package.json (hunk 1 above)
Comment (pmcb55) [reply package.json:3]: Yeah, good point - I'll try to remember to discuss that with Jack as we do the release!

File: src/lib/classes/notification.js (hunk 2 above)
Comment (pmcb55) [reply src/lib/classes/notification.js:464]: Yeah, I reverted this change after chatting with Kevin.
